src/contrib/cluster/Akka.Cluster.Sharding/Serialization/ClusterShardingMessageSerializer.cs

```diff
@@ -699,7 +699,7 @@ private static string ShardIdMessageFromBinary(byte[] bytes)
 
         private IActorRef ResolveActorRef(string path)
         {
-            return system.Provider.ResolveActorRef(path);
```

Comment:
I'm writing docs for the new Akka.NET website on `IActorRef` serialization and it's very disjointed - we use `Serialization.SerializedActorPath` to serialize actors and `ExtendedActorSystem.Provider.DeserializeActorRef`. I wanted to put an access point for both of these functions (actor reference serialization and deserialization) so it makes more sense to end-users trying to handle this problem on their own.